packages/react/src/MinimalFooter/MinimalFooter.features.stories.tsx

```diff
@@ -61,9 +61,10 @@ export const MaximumLinks = () => (
 )
 MaximumLinks.storyName = 'Maximum Links (5)'
 
-export const ButtonLinks = () => (
+export const MixedButtonsAndLinks = () => (
   <MinimalFooter socialLinks={false}>
-    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button one')}>
+    <MinimalFooter.Link>Link one</MinimalFooter.Link>
+    <MinimalFooter.Link as="a" href="#" onClick={() => alert('You have clicked Button one')}>
```

Comment:
@JoshBowdenConcepts I'm still seeing `href` and `onClick` (in the context of a button) being applied at the same time? Is this supposed to be an `a` or a `button`? Given the label is `Button one`, recommend you remove the `as="a" href="#"`.

```suggestion
    <MinimalFooter.Link as="button" onClick={() => alert('You have clicked Button one')}>
```